source/upgrade/v3.txt

```diff
@@ -0,0 +1,114 @@
+.. _csharp-upgrade-v3:
+
+======================
+Upgrade to Version 3.0
+======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, breaking change, releases
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This page describes the changes you might need to make to your
+application when you upgrade the {+driver-short+} to version 3.0.
+
+How to Upgrade
+--------------
+
+Each section of this page lists the potential breaking changes introduced by a driver version.
+To upgrade the {+driver-short+} to version 3.0, follow these steps:
+
+1. Review the :ref:`Compatibility <csharp-compatibility-tables>` page to ensure the new
+   driver version is compatible with the {+mdb-server+} versions your application connects
+   to and the .NET or {+framework+} version your application runs on.
+
+#. If you're using a 2.x version of the {+driver-short+}, upgrade to
+   v{+last-version-2-number+}. To do so, follow the
+   :ref:`v2.x upgrade guide <csharp-upgrade-v2>` guide.
+
+#. Address any breaking changes between the driver version your
+   application uses and your target upgrade version in the
+   :ref:`Breaking Changes by Version <csharp-breaking-changes-v3>` section.
```

Comment:
Q: Since they have to upgrade to v2.9 first, wouldn't this step just be to address the changes listed below? Judging by the Slack thread w/ Rishit, it seemed to me like we will only be showing 3.0 and no minor versions on this page?

Reply:
good point. i didn't think to change that after rishit suggested updating the page title